<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoTypesWidget</class>
 <widget class="QWidget" name="AutoTypesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>103</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Auto fill:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="freeTextCheckBox">
     <property name="text">
      <string>Free text</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="fplCheckBox">
     <property name="text">
      <string>FPL</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="arrCheckBox">
     <property name="text">
      <string>ARR</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chgCheckBox">
     <property name="text">
      <string>CHG</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cnlCheckBox">
     <property name="text">
      <string>CNL</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="depCheckBox">
     <property name="text">
      <string>DEP</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="dlaCheckBox">
     <property name="text">
      <string>DLA</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
